CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## `6.18.1`
```

Comment:
Please leave this as `Recent changes`
That will avoid having to merge PRs in a certain order.
Also, this may be an enhancement since it's adding new functionality to already existing APIs

Reply:
Fixed in recent push :)